signac_dashboard/modules/image_viewer.py

```diff
@@ -35,10 +35,12 @@ def __init__(
         name="Image Viewer",
         context="JobContext",
         template="cards/image_viewer.html",
-        img_globs=["*.png", "*.jpg", "*.gif"],
+        img_globs=None,
```

Comment:
If the goal is to make this argument immutable, we can use a tuple rather than `None`. Tuples are immutable. I would like to keep the default argument something that has a clear value, since it appears in the documentation.
```suggestion
        img_globs=("*.png", "*.jpg", "*.gif"),
```

Reply:
Noted